builtin/update-index.c

```diff
@@ -1098,7 +1099,13 @@ int cmd_update_index(int argc, const char **argv, const char *prefix)
 	argc = parse_options_end(&ctx);
 
 	getline_fn = nul_term_line ? strbuf_getline_nul : strbuf_getline_lf;
+	if (mark_skip_worktree_only && gvfs_config_is_set(GVFS_BLOCK_COMMANDS))
+		die(_("modifying the skip worktree bit is not supported on a GVFS repo"));
```

Comment:
I know we are working to get rid of the pre-command hook but will it have already have ran at this point? What will be the consequences?  Will the post-command still run?  If we are going to get this change in before getting rid of them we need to make sure it will still work with the pre-command.

Reply:
This doesn't change the behavior of the pre/post-command hooks at all so, yes, the pre-command hook will have run.  The post-command hook will still run as well.

This enables us to remove the command blocking for these commands from the pre-command hook - just one step towards being able to get rid of it.